### DISCUSSION TITLE
Can i get transition dipole moment from Delta-SCF with maximium occupation ?

### DISCUSSION THREAD

**Discussion by choconostle** [pyscf/pyscf#2636] [Q&A]
Hello everyone I wrote this issue because I need help with this problem: 

I would like to know if it's possible to get transition dipole moment from a calculation of Delta-SCF with maximium occupation?

**Comment by JangidBhavnesh** [comment]
In $\triangle$-SCF, the wavefunctions of different states are not orthogonal to each other. You would require NOCI (non-orthogonal CI) type formalism to obtain the transition dipole moment. 

You might wanna take a look at #1697

**Comment by sunqm** [comment]
You can use the code like https://github.com/pyscf/pyscf-forge/blob/bb743d16ecf997d935e32a3740f41991ad57882a/pyscf/msdft/noci.py#L152
to compute the density matrix between the two determinants, then the `scf.hf.dip_moment(mol, dm)` for the transition dipole.
